### ISSUE TITLE
취업날개 온라인 이용자가 무료로 대여할 수 있는 방법이 필요함

### ISSUE THREAD

**Issue by 10001if** [opencloset/OpenCloset-Share-Web#153]
서울시의 요청으로 취업날개 이용자가 쿠폰번호를 직접 발급받지 않는 방식으로 예약을 진행하고 있습니다.
때문에 온라인 대여시 쿠폰 번호를 입력할 수 있는 창이 있어도, 취업날개 온라인 이용자는 쿠폰을 알고 있지 못하기 때문에 창에 쿠폰번호를 입력할 수 없습니다.

다른 기관에서 예약 시스템을 만들고 있는데, 어떤 방법으로 온라인 대여를 진행할 수 있는지 잘 모르겠습니다. 우선 오전에 담당 주무관과 통화하였습니다. 정식으로 서비스가 오픈되는 5/22일까지는 온라인 대여를 진행하지 않도록 하였습니다. 정식 오픈 전까지 방법을 고민해 보아야 합니다.

**Comment by aanoaa** [comment]
쿠폰번호를 입력해서 온라인 대여를 할 수 있습니다.
온라인으로 대여하는 절차는 준비가 되어 있는데,

서울시민인지 확인을 어떻게 할 것인지가 어려울 것 같습니다.

**Comment by 10001if** [comment]
@aanoaa 현재 쿠폰번호를 임시로 안내해주고 결제 시점에 쿠폰번호를 넣는 방식으로 변경되었습니다. 완전한 것은 아니나 우선 문제가 해결된 것으로 보입니다. 이슈 닫도록 하겠습니다.

**Comment by aanoaa** [comment]
임시로 해결된 것이기 때문에 다시 열도록 하겠습니다.
좀 더 그레이스풀한 방법을 고민해보도록 하겠습니다.
왜냐면 직접 쿠폰번호를 입력하지 않는게 더 편리하거든요.

**Comment by aanoaa** [comment]
통계문제도 여전히 남아 있습니다.
온라인과 오프라인을 분리해서 보여줄지 어찌할지 고민해봐야 합니다.

**Comment by 10001if** [comment]
@aanoaa 말씀하신 것처럼 쿠폰번호를 입력하지 않는 방법이 더 좋은 방법일 것 같습니다. 통계에 대해서는 온라인과 오프라인이 구분되어 보여지는 것이 필요하다고 생각됩니다.

**Comment by aanoaa** [comment]
취업날개를 이용한 온라인 대여자 수를 알 수 있는 방법이 필요합니다.

**Comment by aanoaa** [comment]
> 통계문제도 여전히 남아 있습니다.
> 온라인과 오프라인을 분리해서 보여줄지 어찌할지 고민해봐야 합니다.

[취업날개 서비스의 방문/미방문자의 통계수치 동기화](https://github.com/opencloset/opencloset/issues/1256) 에서 해결되었습니다.

**Comment by aanoaa** [comment]
2번과 3번을 절차가 번거롭다는 것인데
이러한 아이디어가 필요한 문제들은 시간적 여유가 있을 때 진행하겠습니다.
아이디어가 있으면 적어주세요.
고려해보도록 하겠습니다.

**Comment by 10001if** [comment]
@aanoaa 여기에서 핵심은 쿠폰번호를 입력하지 않고 대여하는 것이라고 생각합니다. 하지만 이 부분에 대해서는 개발적인 아니디어가 필요하다고 생각합니다.. 그래서 저는 쿠폰번호를 입력하지 않는 방법이 없다면. 우선 쿠폰번호를 문자로 보내고 문자에 양도할 수 없다는 내용을 넣고 주문 마지막에 쿠폰번호를 입력하라고 설명을 포함시키면 어떨까 싶은데요.. 쿠폰번호가 공유되면 문제가 발생할까요..?

**Comment by 10001if** [comment]
물론 이 역시 그레이스한 방법은 아닙니다. 다만, 이용자가 이용하지 못하는 문제를 해결하기 위한 또 하나의 방법입니다.

**Comment by aanoaa** [comment]
> 쿠폰번호가 공유되면 문제가 발생할까요..?

지금도 쿠폰번호는 공유될 수 있습니다.
단지 SMS 를 더 보내는 기능일 뿐입니다.

**Comment by 10001if** [comment]
@aanoaa 맞습니다. 하지만 현재는 쿠폰번호를 사진을 찍거나 캠쳐하는 등의 절차가 동반되지 않으면 애초에 대여를 할 수 없습니다.
